### PR TITLE
Add Job summary

### PR DESCRIPTION
**Pull request by gastaldi**
- Fixes https://github.com/smallrye/smallrye-release/issues/11

Here is a sample output: https://github.com/quarkiverse/quarkus-prettytime/actions/runs/10821548543#summary-30556126806

### CODE REVIEW

File: .github/workflows/perform-release.yml [hunk 1]
```diff
@@ -137,7 +137,7 @@ jobs:
           # Wait for the deploy-artifacts.yml workflow to start
           sleep 3
```
Comment (cescoffier) [inline .github/workflows/perform-release.yml:138]: Is it really safe?

File: .github/workflows/perform-release.yml [hunk 2]
```diff
@@ -137,7 +137,7 @@ jobs:
...
           # Get the URL of the release workflow (MUST match the workflow's run-name)
-          SUMMARY_URL=$(gh run list --workflow deploy-artifacts.yml -R quarkiverse/quarkiverse-release --json name,url -q ".[] | select(.name == \"${WORKFLOW_RELEASE_RUN_NAME}\") | .url" --limit 1)
+          SUMMARY_URL=$(gh run list --workflow deploy-artifacts.yml -R smallrye/smallrye-release --json name,url -q ".[] | select(.name == \"${WORKFLOW_RELEASE_RUN_NAME}\") | .url" --limit 1)
```
Comment (cescoffier) [inline .github/workflows/perform-release.yml:140]: Do you have an example somewhere? 

How does it work if the process is not done yet.

**Review by cescoffier — CHANGES_REQUESTED** [review]

File: .github/workflows/perform-release.yml (hunk 2 above)
Comment (gastaldi) [reply .github/workflows/perform-release.yml:140]: This returns the queued workflow run URL, it doesn’t matter its state, you can monitor the logs related to this deployment once you click on it

File: .github/workflows/perform-release.yml (hunk 1 above)
Comment (gastaldi) [reply .github/workflows/perform-release.yml:138]: Not ideal, but it does the job, at least until https://github.com/cli/cli/issues/4001 is implemented

**Review by radcortez — APPROVED** [review]

**Pull request merged**